lib/version_utils.pm

```diff
@@ -223,7 +223,7 @@ sub is_microos {
     elsif ($filter =~ /\d\.\d\+?$/) {
         # If we use '+' it means "this or newer", which includes tumbleweed
         return ($filter =~ /\+$/) if ($version eq 'Tumbleweed' && $type eq 'opensuse');
-        return check_version($filter, $version, qr/\d\.\d/);
+        return check_version($filter, $version, qr/\d{2,}\.\d/);
```

Comment:
Doesn't this mean "at least two digits"? If so, it would fail for SUSE MicroOS (5)

Reply:
I missed it, sorry. Fixed now.

Reply:
Last VR with this new change passing: http://fromm.arch.suse.de/tests/326